File: src/main/tests/pythonGenerated/max_element.py
import typing

def findMax(arr, n) -> int:
    maxVal: int = arr[0]
    for i in range(1, n):
        if arr[i] > maxVal:
            maxVal = arr[i]
    return maxVal

def main() -> int:
    arr: list[int] = [10, 25, 30, 42, 15]
    n: int = len(arr)
    print("Maximum element is ", findMax(arr,n), sep="")
    print()
    return 0

if __name__ == "__main__":
    main()

File: src/main/tests/input/max_element.cpp
#include <iostream>
#include <vector>

int findMax(std::vector<int>& arr, int n) {
    int maxVal = arr[0];
    for (int i = 1; i < n; i++) {
        if (arr[i] > maxVal)
            maxVal = arr[i];
    }
    return maxVal;
}

int main() {
    std::vector<int> arr = {10, 25, 30, 42, 15};
    int n = arr.size();
    std::cout << "Maximum element is " << findMax(arr, n) << std::endl;
    return 0;
}
